codex-rs/tui/src/bottom_pane/approval_overlay.rs

```diff
@@ -364,14 +431,29 @@ fn render_risk_lines(risk: &SandboxCommandAssessment) -> Vec<Line<'static>> {
 
 #[derive(Clone)]
 enum ApprovalVariant {
-    Exec { id: String, command: Vec<String> },
-    ApplyPatch { id: String },
+    Exec {
+        id: String,
+        command: Vec<String>,
+    },
+    ApplyPatch {
+        id: String,
+    },
+    Elicitation {
+        server_name: String,
+        request_id: RequestId,
+    },
+}
+
+#[derive(Clone, Copy)]
```

Comment:
Ultimately, `ElicitationDecsion` will have to have more fields because the request could have a whole form the user is supposed to fill out, so it probably won't be able to be `Copy` in the near future.